### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by laikrai** [Pizzabelly/EasyRP#140]
Hi, when i do the command this error pops up!
![image](https://user-images.githubusercontent.com/71832599/100851924-1ce58d00-3486-11eb-87d6-a474119d9102.png)
[meson-log.txt](https://github.com/Pizzabelly/EasyRP/files/5628066/meson-log.txt)
Can you help me since i don't understand what am i doing...

**Comment by bhavyagosai** [comment]
You really want to build it from source or just wanna use it?

**Comment by laikrai** [comment]
> You really want to build it from source or just wanna use it?

I want to use it

**Comment by bhavyagosai** [comment]
Yo then just download and run the [executable from the releases!](https://github.com/Pizzabelly/EasyRP/releases)

**Comment by bhavyagosai** [comment]
Also check this [repo](https://github.com/bBSempai/EasyRP) if you wanna automate it!

**Comment by laikrai** [comment]
but i cant run easyrp for some reason... It's something called shared library it says

> Yo then just download and run the [executable from the releases!](https://github.com/Pizzabelly/EasyRP/releases)

**Comment by bhavyagosai** [comment]
What did you do can you provide ss

**Comment by laikrai** [comment]
i downloaded .tar.gz and then i cant open easyrp

**Comment by bhavyagosai** [comment]
You did follow the instructions in the [readme.txt](https://github.com/Pizzabelly/EasyRP/files/5629275/readme.txt) and configure the config.ini right?

**Comment by laikrai** [comment]
yes and when i open easyrp i cant open it i have to select a app 
![image](https://user-images.githubusercontent.com/71832599/100874206-82487680-34a4-11eb-8653-6894ec3f5dd0.png)
It looks liek i need some sort of app

**Comment by bhavyagosai** [comment]
Ahh you using LINUX
okay so in the directory with the config file and other files open terminal and run these commands
```chmod +x easyrp```
```./easyrp```
It should work

**Comment by laikrai** [comment]
![image](https://user-images.githubusercontent.com/71832599/100877778-8dea6c00-34a9-11eb-8273-a3bfb3b96235.png)

**Comment by bhavyagosai** [comment]
Extract the tar.gz and then run the command

**Comment by laikrai** [comment]
> Extract the tar.gz and then run the command

i did

**Comment by bhavyagosai** [comment]
This is the only method. Just check if the name of the files you are using havent been changed by you or anything because thats the only reason a directory does not exist error can come. Also while typing ./easyrp press tab to check if the terminal autocompletes the directory. If it doesnt then that directory doesnt exist

**Comment by laikrai** [comment]
> This is the only method. Just check if the name of the files you are using havent been changed by you or anything because thats the only reason a directory does not exist error can come. Also while typing ./easyrp press tab to check if the terminal autocompletes the directory. If it doesnt then that directory doesnt exist

i tried everything over again and this error happend
![image](https://user-images.githubusercontent.com/71832599/101247477-37d02f80-371a-11eb-8904-05bb9fa391c1.png)

**Comment by bhavyagosai** [comment]
damn boyah ask the author ig @Pizzabelly idk I havent built it :((

**Comment by dzlandis** [comment]
Getting the same error. Tried everything :( 

![image](https://user-images.githubusercontent.com/49926644/115947572-69555780-a47d-11eb-987c-9b44cec2141a.png)
